api/src/handlers.rs

```diff
@@ -1,4 +1,4 @@
-// Copyright 2016 The Grin Developers
+// Copyright 2017 The Grin Developers
```

Comment:
Thanks for remembering to update the copyright. By the time I remember it will be 2018...

Reply:
I'm never 100% sure if these _should_ be updated?